src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheClient.cs

```diff
@@ -41,11 +41,11 @@ public RedisCacheClient(AzureContext context)
         }
         public RedisCacheClient() { }
 
-        public RedisResourceWithAccessKey CreateOrUpdateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
+        public RedisResource CreateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
```

Comment:
These are not breaking changes from customer side. These are cause by changes in swagger changes done by SDK team.